jobfair-frontend/src/containers/CompanyRegisterForm/CompanyRegisterForm.container.jsx

```diff
@@ -0,0 +1,197 @@
+import { Button, Card, Form, Modal, Steps, Typography } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PATH } from '../../constants/Paths/Path';
+import { createCompanyAPI } from '../../services/jobhub-api/CompanyControllerService';
+import { faBuilding, faUserShield } from '@fortawesome/free-solid-svg-icons';
+import { registerCompanyAPI } from '../../services/jobhub-api/CompanyEmployeeControllerService';
+import { useHistory } from 'react-router-dom';
+import CompanyManagerRegisterFormComponent from '../../components/forms/CompanyManagerRegisterForm/CompanyManagerRegisterFormComponent';
+import CreateCompanyFormComponent from '../../components/forms/CreateCompanyForm/CreateCompanyForm.component';
+import React, { useState } from 'react';
+import RegisterSuccessContentComponent from '../../components/commons/RegisterSuccessContent/RegisterSuccessContent.component';
+
+const CompanyRegisterFormContainer = () => {
+  const [form] = Form.useForm();
+  const [currentStep, setCurrentStep] = useState(0);
+  const history = useHistory();
+
+  const registerCompanyMapper = (values) => ({
+    name: values.name,
+    companyEmail: values.companyEmail,
+    taxId: values.taxId,
+    sizeId: values.sizeId,
+    subCategoriesIds: values.subCategoriesIds
+  });
+
+  const handleRegisterCompany = async (values) => {
+    const companyBody = registerCompanyMapper(values);
+    try {
+      return await createCompanyAPI(companyBody);
+    } catch (e) {
+      Modal.error({
+        title: 'Register company failed !',
+        width: '30rem',
+        height: '40rem',
+        closable: true,
+        maskClosable: true,
+        content: (
+          <>
+            <Typography.Text strong>{e.response.data.message}</Typography.Text>
+          </>
+        )
+      });
+    }
+  };
+
+  const handleOnFinish = async () => {
+    try {
+      const res = await handleRegisterCompany(form.getFieldsValue(true));
+      const responseManager = await handleRegisterCompanyManager(form.getFieldsValue(true), res.data.id);
+      if (res.status === 200 && responseManager.status === 201) {
```

Comment:
What happen if request has error and res and responseManager is undefined?

Reply:
handleRegisterCompany and handleRegisterManager had catch errors, so there is no need to catch errors here.